ro-base.obo

```diff
@@ -994,6 +999,7 @@ subset: ro-eco
 property_value: IAO:0000117 "David Osumi-Sutherland" xsd:string
 property_value: IAO:0000118 "d" xsd:string
 property_value: IAO:0000118 "during" xsd:string
+property_value: seeAlso Happens:during
```

Comment:
?

Reply:
Fixed in the commit [c0ecc1](https://github.com/oborel/obo-relations/pull/728/commits/c0ecc1df96a957bd979bc98214b85b855e6818ad)